### PR TITLE
DG in zzy features

### PR DESCRIPTION
**Pull request by zzybluebell**

### CODE REVIEW

File: docs/team/zzybluebell.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Given below are my contributions to the project.
 <bc>
 
   * **New Feature**: Redeem points [\#103](https://github.com/AY2122S1-CS2103T-F12-4/tp/pull/103)
-  * **What it does**: Redeemed as amounts of cash deduction to a member.
+  * **What it does**: Redeemed as discounts to a member.
   * **Justification**: 
     * The feature is essentials, and the priority is medium.
     * The point accumulation is similar to credit.
```
Comment (Morrow1ndy) [inline docs/team/zzybluebell.md:54]: LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -331,10 +331,37 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+
+7. Through the `ParseUtil`, it will get default `timestamp`, `credit`, `point`, `tagList`, `transactionList` and `reservationList`.
+
+7. Using the obtained `name`, `phone`,`email`, `address`, `timestamp`, `credit`,`point`, `tagList`, `transactionList` and `reservationList`. a new instance of `Member` is created.
+
+8. Using a new instance `Member`, a new instance of `AddMemberCommand` is created and returned to `ezFoodieParser` which in turn returns it to `LogicManager`.
+
+9. `LogicManager` calls the `AddMemberCommand#execute(Model)` method.
+
+10.  Lastly, `AddMemberCommand` creates a new instance of `CommandResult` with a success message, and returns it to Logic Manager.
```
Comment (Morrow1ndy) [inline docs/DeveloperGuide.md:355]: `LogicManger` instead?

**Review by Morrow1ndy — COMMENTED** [review]
Minor changes

**Review by mukundrs — APPROVED** [review]
LGTM

**Pull request merged**